Common Example code/PaymentsController.swift

```diff
@@ -184,17 +184,19 @@ internal final class PaymentsController {
     }
 
     private func performPaymentDetails(with data: ActionComponentData) {
-        let request = PaymentDetailsRequest(details: data.details, paymentData: data.paymentData)
+        let request = PaymentDetailsRequest(details: data.details,
+                                            paymentData: data.paymentData,
+                                            merchantAccount: Configuration.merchantAccount)
         apiClient.perform(request, completionHandler: paymentResponseHandler)
     }
 
     private func paymentResponseHandler(result: Result<PaymentsResponse, Error>) {
         switch result {
         case let .success(response):
             if let action = response.action {
-                currentComponent?.stopLoading()
                 handle(action)
             } else {
+                currentComponent?.stopLoading()
```

Comment:
Not needed, since finish also calls `currentComponent?.stopLoading()`

Reply:
👍 , will remove it.